code/modules/antagonists/heretic/magic/rust_wave.dm

```diff
@@ -19,7 +19,7 @@
 
 /obj/effect/proc_holder/spell/cone/staggered/entropic_plume/do_turf_cone_effect(turf/target_turf, level)
 	. = ..()
-	target_turf.rust_heretic_act()
+	target_turf.rust_heretic_act(usr)
```

Comment:
Have you tested this / are you sure that `usr` is set correctly for this proc? You may need to adjust cone spells a bit if not.

Reply:
Yeah this works fine.